library/alloc/Cargo.toml

```diff
@@ -10,6 +10,7 @@ edition = "2021"
 
 [dependencies]
 core = { path = "../core" }
+safety = { path = "../contracts/safety" }
```

Comment:
This change now is no longer necessary as the same has already been added in #121 two lines below.